sdk/cpp/core/src/path/netconf_session.cpp

```diff
@@ -489,8 +489,9 @@ static string get_filter_payload(path::Rpc & ydk_rpc)
 {
     auto entity = ydk_rpc.get_input_node().find("filter");
     if(entity.empty()){
-        YLOG_ERROR("Failed to get entity node.");
-        throw(YInvalidArgumentError{"Failed to get entity node"});
+        YLOG_WARN("Failed to get filter node for RPC; it then will be skipped.");
```

Comment:
Why is this changed to warning?

Reply:
It is not an error when filter is not present. In this case the whole device configuration will be fetched.